Ginger/Ginger/SourceControl/SourceControlProjectsPage.xaml.cs

```diff
@@ -109,7 +124,7 @@ private void Bind()
             GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(SourceControlUserTextBox, TextBox.TextProperty, mSourceControl, nameof(SourceControlBase.SourceControlUser));
             GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(SourceControlPassTextBox, TextBox.TextProperty, mSourceControl, nameof(SourceControlBase.SourceControlPass));
             GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(txtConnectionTimeout, TextBox.TextProperty, mSourceControl, nameof(SourceControlBase.SourceControlTimeout));
-            UpdateTimeoutVisibility();
+            UpdateTimeoutAndBranchPanelVisibility();
```

Comment:
call the function something like "SetConfigurationsVisibility()" to be more generic